src/main/scala/cromwell/binding/package.scala

```diff
@@ -13,5 +13,7 @@ import cromwell.binding.values.WdlValue
  */
 
 package object binding {
+  type Wdl = String
```

Comment:
Maybe I'll be downvoted into oblivion, though isn't it clear enough to say `String wdlSource` wherever it's used?  However if we have to choose one I'd say `WdlSource` too